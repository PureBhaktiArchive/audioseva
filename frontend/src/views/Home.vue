/*
 * sri sri guru gauranga jayatah
 */
<template>
  <v-container fluid>
    <v-layout row wrap>
      <v-flex xs12 class="text-xs-center" mt-5>
        <h1>Welcome to Audio Seva Backend</h1>
        <v-btn to="sqr" color="primary">SQR</v-btn>
        <v-btn to="se" color="primary">SE</v-btn>
      </v-flex>
    </v-layout>
  </v-container>
</template>

<script>
export default {
  name: "Home"
};
</script>
