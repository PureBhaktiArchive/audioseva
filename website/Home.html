<h2>The Objective</h2>

<p>
  To systematically organize the archives (audio &amp; video) of Srila Gurudeva.
</p>

<h2>Background</h2>

<p>Over the years more than 10000 lectures of Srila Gurudeva were recorded.</p>

<p>
  Since the recording, especially in the earlier years, was not undertaken
  systematically, multiple challenges remain to be tackled in the archives, such
  as:&nbsp;
</p>

<ul>
  <li>Undated lectures&nbsp;</li>
  <li>Fused lectures&nbsp;</li>
  <li>Duplicates</li>
  <li>Impaired sound quality, etc.&nbsp;</li>
</ul>

<p>
  To go through the Audio Seva guidelines in detail,
  <a href="/audioseva/guidelines">click here</a>.
</p>

<p>&nbsp;</p>

<div class="embed-responsive embed-responsive-16by9">
  <iframe
    allow="autoplay; encrypted-media"
    allowfullscreen=""
    class="embed-responsive-item"
    src="https://www.youtube.com/embed/GyxUJjqzw3Q?rel=0"
  ></iframe>
</div>

<p>&nbsp;</p>

<div class="embed-responsive embed-responsive-16by9">
  <iframe
    allow="encrypted-media"
    allowfullscreen=""
    class="embed-responsive-item"
    src="https://www.youtube.com/embed/ANtLndkBvOk?rel=0"
  ></iframe>
</div>

<p>&nbsp;</p>

<div class="row">
  <div class="col-sm-4 col-md-offset-4">
    <a
      class="btn btn-primary btn-lg btn-block"
      href="/audioseva/form/registration"
      style="background: linear-gradient(#bb625c, #9e2017 60%, #bb625c);"
      type="button"
      >Register now!</a
    >
  </div>
</div>
