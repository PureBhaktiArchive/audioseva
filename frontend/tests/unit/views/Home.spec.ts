import { shallowMount } from "@vue/test-utils";
import Home from "@/views/Home.vue";

describe("Home.vue", () => {
  it("should render", () => {
    const wrapper = shallowMount(Home);
    expect(wrapper.element).toMatchSnapshot();
  });
});
