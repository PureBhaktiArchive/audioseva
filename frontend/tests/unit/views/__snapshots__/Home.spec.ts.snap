// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.vue should render 1`] = `
<v-container
  fluid=""
>
  <v-layout
    row=""
    wrap=""
  >
    <v-flex
      class="text-xs-center"
      mt-5=""
      xs12=""
    >
      <h1>
        Welcome to Audio Seva Backend
      </h1>
       
      <v-btn
        color="primary"
        to="sqr"
      >
        SQR
      </v-btn>
    </v-flex>
  </v-layout>
</v-container>
`;
